fireants/interpolator/grid_sample.py

```diff
@@ -23,6 +23,29 @@
 import logging
 logger = logging.getLogger(__name__)
 
+
```

Comment:
duplicate code - can you remove the corresponding code from `fused_grid_sample.py` and change all corresponding references?